tests/io/test_eleanorlite.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from astropy.io import fits
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from lightkurve import search_lightcurve
+from lightkurve.io.eleanorlite import read_eleanorlite_lightcurve
+from lightkurve.io.detect import detect_filetype
+
+
+@pytest.mark.remote_data
+def test_eleanor_lite():
+    """Can we read in GSFC-ELEANOR-LITE light curves?"""
+    url = "https://mast.stsci.edu/api/v0.1/Download/file?uri=mast:HLSP/gsfc-eleanor-lite/s0001/0000/0001/4006/8425/hlsp_gsfc-eleanor-lite_tess_ffi_s0001-0000000140068425_tess_v1.0_lc.fits"    
+    with fits.open(url, mode="readonly") as hdulist:
+        # Can we auto-detect a QLP file?
```

Comment:
Nitpick: the comment should be updated from QLP to GSFC-ELEANOR-LITE

Reply:
Thank you, it's fixed now.